### ISSUE TITLE
Can't set avatar : "TigaseSwift.PubSubError error 1."

### ISSUE THREAD

**Issue by eerielili** [tigase/siskin-im#161]
**Describe the bug**
When my contact tries to change its profile picture within Siskin, he's met with the error _"User avatar publication failed. The operation couldn't be completed. (TigaseSwift.PubSubError error 1.)"_

**To Reproduce**
Described in [the documentation](https://docs.tigase.net/tigase-ios/master-snapshot/Tigase_Swift_Guide/html/#_accounts), tap _"Choose avatar"_ and select a picture

**Expected behavior**
The selected image is the avatar that'll be set.

**Screenshots**
![image](https://user-images.githubusercontent.com/81772782/148654227-7f39cbb2-a7e5-4788-9163-0db000f4a7e7.png)

**Details (please complete the following information):**
 - Siskin Version: 7.0.1
 - iOS version : 15.1
 - iPhone model : X

**Additional context**
I own the server side of things. It's a prosody server version 0.11.11 on Debian Buster. Modules enabled related to avatar are the following:

- pep_simple
- http_pep_avatar
- pep_vcard_avatar

Prosody modules are kept up to date with hg.

I digged a bit, and [it seems you're using XEP-0060 for PubSub](https://docs.tigase.net/tigase-pubsub/master-snapshot/Tigase_PubSub_Guide/html/#PubSubComp), is that correct for Siskin as well ?

Thanks.

**Comment by hantu85** [comment]
I guess you are referring to the issue discussed in the Tigase support channel and already fixed in 3b62660620da1b3e2aca3458871aae5089774a5c

The issue was caused by using 180x180 PNG for avatars which were rejected by ejabberd (and most likely by Prosody as well) due to the large size of the stanza.

This fix is scheduled to be released as a part of the next version (7.1).

**Comment by eerielili** [comment]
> I guess you are referring to the issue discussed in the Tigase support channel and already fixed in [3b62660](https://github.com/tigase/siskin-im/commit/3b62660620da1b3e2aca3458871aae5089774a5c)

Yeah I have seen this the other day and was wondering if it got any link to this issue. Thanks for confirming it ! xmpp rocks
